src/booleans.js

```diff
@@ -1,61 +1,127 @@
 function negate(a) {
-  // your code here
+  if(a === true){
```

Comment:
Your code works, so good stuff. However, you don't need to use if else statements in this function. You can use the `!` operator and do something like `return !a` because `!a` resolves to either true or false.